src/lib/txn.js

```diff
@@ -209,6 +181,9 @@ const isTransactionConfirmed = async (web3, txHash) => {
   return receipt !== null;
 };
 
+const sendTransactionsAndAwaitConfirm = async (web3, signedTxs, usedTank) =>
+  Promise.all(signedTxs.map(tx => sendSignedTransaction(web3, tx, usedTank)));
+
```

Comment:
This resolves on-submit, not on-confirm, as per the multiple `resolve` calls in `sendSignedTransaction`. Fine with punting on that until the refactoring of this lib, but we shouldn't name this misleadingly in the meantime.

Reply:
ohhh that's why the errors aren't being caught. this code is (attempting to be) semantically the same as the previous version, so update this to wait for each of those transactions